mreg/api/permissions.py

```diff
@@ -72,7 +72,9 @@ class IsInRequiredGroup(IsAuthenticated):
     Allows only access to users in the required group.
     """
 
-    def has_permission(self, request, view):
+    # Note that required_user_groups is not used internally and requires a
+    # settings change for proper testing.
```

Comment:
This is old code and can probably be removed instead. Was from the time of internal testing we required users to be part of a group. Still usefull feature though.

Reply:
It's a bit tricky. It's code that isn't really supported without a change to settings, which makes testing quite annoying. I made an effort to try to mock settings, but... Django didn't really enjoy that very much. On the other hand we have no way of knowing if a third party at this point relies on the feature...

Reply:
All traces of required_user_groups is removed via https://github.com/unioslo/mreg/pull/484/commits/6dc47fc09b822bdb75b13ab62fc4e10240129d3f